cmd/root.go

```diff
@@ -20,6 +19,8 @@ const (
 	manifestPathArgName = "manifest-path"
 )
 
+// Version is the CLI tool version, this should be overridden at build time for anything to be distributed
+var Version = "dev"
```

Comment:
What's the motivation on removing version.go and defining this variable instead?

Reply:
Good question, and I'm not 100% sold on this yet, first time I'm trying it like this for a Go CLI tool, but I think it might prove to be a good model for setting a version at build time (https://github.com/rockholla/loftsman/blob/feature/go-1.16/.github/workflows/release.yml#L94), e.g.:

```
go build -o loftsman -ldflags "-X 'github.com/Cray-HPE/loftsman/cmd.Version=1.0.4'"
```

My hope is that versioning can be more hands off this way, we only care about git tagging to release a version. One strange point that remains is using the `go get github.com/Cray-HPE/loftsman` way of installing will install with the version `dev` by default. I did call out how to use go get with `ldflags` to set some version as well. Not sure if that will end up being weird, but figured we'd give this a shot. Happy to continue to talk about it, alternatives, etc.